### PR TITLE
BugFix(Notification) - Service notification title and message issue fix

### PR DESCRIPTION
**Pull request by manojava-gk**

### CODE REVIEW

File: cx-portal/src/components/pages/NotificationCenter/NotificationItem.tsx [hunk 1]
```diff
@@ -154,6 +155,10 @@ const NotificationConfig = ({ item }: { item: CXNotificationContent }) => {
       return <NotificationContent item={item} navlinks={['appoverview']} />
     case NotificationType.TECHNICAL_USER_CREATION:
       return <NotificationContent item={item} navlinks={['technicaluser']} />
+    case NotificationType.SERVICE_REQUEST:
+      return (
+        <NotificationContent item={item} navlinks={['servicesubscription']} />
```
Comment (oyo) [inline cx-portal/src/components/pages/NotificationCenter/NotificationItem.tsx:160]: Please use `PAGES.SERVICESUBSCRIPTION` from here: https://github.com/catenax-ng/tx-portal-frontend/blob/main/cx-portal/src/types/Constants.ts#L92
maybe also change for the other strings...

**Review by oyo — CHANGES_REQUESTED** [review]
noticed that we can use already defined constants

File: cx-portal/src/components/pages/NotificationCenter/NotificationItem.tsx (hunk 1 above)
Comment (manojava-gk) [reply cx-portal/src/components/pages/NotificationCenter/NotificationItem.tsx]: Done

**Review by oyo — APPROVED** [review]
looks good now

**Pull request merged**